src/tsf/TfCatUtil.cpp

```diff
@@ -37,7 +37,7 @@ CicCategoryMgr::CicCategoryMgr()
 CicCategoryMgr::~CicCategoryMgr()
 {
     if (m_pcat) {
-        m_pcat.Release();
+        m_pcat.reset();

```

Comment:
I've no idea why the original code calls `Release` here, it shouldn't be necessary.